rhba-dev-env/ansible/group_vars/all/vars_file.yml

```diff
@@ -0,0 +1,59 @@
+---
+### environment vars ###
+workspace_home:  /home/vagrant/workspace
+# https://github.com/ansible/ansible/issues/54855
+ansible_python_interpreter: /usr/bin/python3
+# same "ip" from config.yml (Vagrantfile on vagrant-ocp-bsig)
+vm_ip: 192.168.0.40
+# same "name" from config.yml (Vagrantfile)
+vm_name: workstation
+
+### kerberos config ###
+kerberos_username: mmagnani
+kerberos_password: my_kerberos_password
+
+### useful scripts ###
+build_overrides_version: '7.4.0'
+build_overrides_product: 'rhpam'
+build_overrides_nightly: '20190724'
+build_product: 'rhpam-all'
+
+git_branch_target: remotes/origin/7.4.x
```

Comment:
why origin instead upstream/master?

Reply:
@spolti Does it make a difference that this is for the developer's use of builds?

Reply:
at least my self, I always forget to rebase my local branch.

Reply:
@spolti ack I will put the "upstream" as default.